associations/wmt.md

```diff
@@ -189,5 +189,7 @@ There are different types of direct assessment.
 - Bilingual: Human raters see the system input and output.
 - Reference-based: Human raters see the system output and a reference output.
 
+For WMT22, a combination of direct assessment and **scalar quality metric** was used for the evaluation of out-of-english and non-english translation pairs.
+In direct assessment with scalar quality metric, humans also rate each source segment and translated segment from a document on a six-point scale.
```

Comment:
Perhaps we should insert the description of **scalar quality metric** in the [human evaluation article](https://machinetranslate.org/human-evaluation-metrics), instead of here.